cmd/sync.go

```diff
@@ -140,6 +140,11 @@ func (raw *rawSyncCmdArgs) cook() (cookedSyncCmdArgs, error) {
 		azcopyScanningLogger.CloseLog()
 	})
 
+	// if no logging, set this empty so that we don't display the log location
+	if azcopyLogVerbosity == common.LogNone {
```

Comment:
When log level is none, do we also not produce a log file at all?

Reply:
yeah correct. We dont create the log file at all.